docs/docs/joining-a-testnet.md

```diff
@@ -1,58 +1,53 @@
 # Joining a testnet
 
-In order to join the test network you will need to have the correct version of the fetchd ledger available on your system. For users who just want to connect to the network we recommend that you use the docker images. A number of
-the setup steps have been automated to it is very quick to get started.
-
-Alternatively if you plan to run a validator node then it will make more sense in the long term for you are familiar with operating a local version of the software.
-
-## Using the docker image
-
-Much more information is available from the [Docker Images](../docker-images/) pages however, to join a desired network you can simply run the docker images with the following command:
-
-    docker run -e MONIKER=<insert node name here> -e NETWORK=<network name> fetchai/fetchd:0.5
-
-**Beacon World Example**
-
-To connect to the beacon world testnet users would simply need to run the following command
-
-	docker run -e MONIKER=my-first-fetch-node -e NETWORK=beaconworld fetchai/fetchd:0.5
+In order to join the test network you will need to have the correct version of the fetchd ledger available on your system. 
 
 ## Using a local version
 
-Assuming that you have following the [installation guide](../building/). You should now have `fetchd` and `fetchcli` successfully installed in your path. You can check this with the following command:
+Assuming that you have following the [installation guide](../building/). You should now have `fetchd` successfully installed in your path. You can check this with the following command:
 
 ```bash
 which fetchd
-which fetchcli
 ```
 
 You can also verify that you are running the correct version for the [network](../networks/).
 
 ```bash
 fetchd version
-fetchcli version
 ```
 
-### Configuring the client `fetchcli`
-
+### Configuring the client fetchd
 In general to configure the CLI to point at a given network it needs as a minimum the following configuration values
 
 ```bash
-fetchcli config chain-id <chain-id>
-fetchcli config trust-node false
-fetchcli config node <rpc url>
+fetchd config chain-id <chain-id>
+fetchd config node <rpc url>
 ```
+  
+### Stargate example
 
-**Beacon World Example**
+In the case of the Stargate network this would be as follows:
 
-In the case of the beacon world network this would be as follows:
+```bash
+fetchd config chain-id stargateworld-1
+fetchd config node https://rpc-stargateworld.fetch.ai:443
+```
+  
+# now the following will still work the same as above:
 
 ```bash
-fetchcli config chain-id beaconworld-3
-fetchcli config trust-node false
-fetchcli config node https://rpc-beaconworld.fetch.ai:443
+fetchd query staking validators 
+fetchd tx bank send account1 account2 1atestfet 
 ```
 
+And few extra bits:
+
+```bash
+fetchd config # display current config
+cat ~/.fetchd/config/client.toml # this is the file where the config is written
+```
+
+
```

Comment:
```suggestion
```
This doesn't really belong to "joining a testnet", more on a general CLI usage document